[2023-05-19 15:25:34,932] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-05-19 15:25:34,939] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-05-19 15:25:34,939] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 15:25:34,939] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-19 15:25:34,939] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 15:25:34,954] {taskinstance.py:1377} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2023-05-14 00:00:00+00:00
[2023-05-19 15:25:34,956] {standard_task_runner.py:52} INFO - Started process 3301 to run task
[2023-05-19 15:25:34,963] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'TwitterDAG', 'twitter_datascience', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpoc5gf973', '--error-file', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmp67sr68ch']
[2023-05-19 15:25:34,966] {standard_task_runner.py:80} INFO - Job 4: Subtask twitter_datascience
[2023-05-19 15:25:35,035] {task_command.py:370} INFO - Running <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [running]> on host MacBook-Air-de-Mariana-2.local
[2023-05-19 15:25:35,106] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=TwitterDAG
AIRFLOW_CTX_TASK_ID=twitter_datascience
AIRFLOW_CTX_EXECUTION_DATE=2023-05-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-14T00:00:00+00:00
[2023-05-19 15:25:35,114] {base.py:68} INFO - Using connection ID 'twitter_default' for task execution.
[2023-05-19 15:25:35,116] {twitter_hook.py:34} INFO - URL: https://labdados.com/2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2023-05-14T00:00:00.00Z&end_time=2023-05-15T00:00:00.00Z
[2023-05-19 15:25:35,757] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=TwitterDAG, task_id=twitter_datascience, execution_date=20230514T000000, start_date=20230519T182534, end_date=20230519T182535
[2023-05-19 15:25:35,787] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-19 15:25:35,810] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-19 18:17:31,432] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-05-19 18:17:31,444] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-05-19 18:17:31,444] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 18:17:31,444] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-19 18:17:31,445] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 18:17:31,471] {taskinstance.py:1377} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2023-05-14 00:00:00+00:00
[2023-05-19 18:17:31,476] {standard_task_runner.py:52} INFO - Started process 9308 to run task
[2023-05-19 18:17:31,490] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'TwitterDAG', 'twitter_datascience', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpupil6ere', '--error-file', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmprbhu8ayz']
[2023-05-19 18:17:31,493] {standard_task_runner.py:80} INFO - Job 32: Subtask twitter_datascience
[2023-05-19 18:17:31,575] {task_command.py:370} INFO - Running <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-14T00:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2023-05-19 18:17:31,670] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=TwitterDAG
AIRFLOW_CTX_TASK_ID=twitter_datascience
AIRFLOW_CTX_EXECUTION_DATE=2023-05-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-14T00:00:00+00:00
[2023-05-19 18:17:31,681] {base.py:68} INFO - Using connection ID 'twitter_default' for task execution.
[2023-05-19 18:17:31,688] {twitter_hook.py:34} INFO - URL: https://labdados.com/2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2023-05-14T00:00:00.00Z&end_time=2023-05-15T00:00:00.00Z
[2023-05-19 18:17:32,271] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=TwitterDAG, task_id=twitter_datascience, execution_date=20230514T000000, start_date=20230519T211731, end_date=20230519T211732
[2023-05-19 18:17:32,319] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-19 18:17:32,371] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
